docs/personal_data_handling.md

```diff
@@ -0,0 +1,56 @@
+# Data handling in PMM - FAQ
+
+
+The following questions are being answered related to personal and confidential data handling in PMM:
+
+1. Which type of data is transmitted?
+
+      |**Data collection source**                                       | **Data collected** |
+      | --------------------------------------------------------------- | ------------------------------------------------------
+      | DB host to PMM                                                  | Database performance metrics <br/> SQL query examples for query analytics (optional).
+      | PMM to DB Host                                                  | DSN and credentials for database access. A separate DB user is used (read-only) to access the database.
+      | DB Host to Cloud S3                                             | Database backup - optional if PMM Administrator configures it.
+      | PMM Server to Percona Cloud                                     | Telemetry data is collected. </br/> PMM Server collects varying amounts of data from version to version. <br/> No personal or confidential information is collected.
+
+
+2. Where is the data transmitted?
+
+    All data gathered from the DB Host is transmitted to the PMM Server. It is possible to transmit DB backups to Cloud S3 storage (optional). 
+
+    PMM Administrator defines which Cloud S3 storage is used as a backup storage.
+
+3. What is the purpose and nature of data processing?
+
+    As per our [Privacy Policy](https://www.percona.com/privacy-policy), the data collection purposes are to provide the services and product enhancements.
+
+    Although, PMM does not collect nor transfer personal data explicitly, in case query analytics is enabled and query examples collection is not disabled, we gather SQL query examples with real data and personal data may appear there if it is stored in DB.  All QAN data always remains within the PMM server, and is never transmitted anywhere else.
+
+4. What is the frequency and volume of processed data?
+
+    Data is gathered each minute. In case Query Analytics is enabled and SQL query examples are gathered, we don't use any special processing for personal or confidential data. PMM server has no clue about the meaning of the data inside the SQL query. 
```

Comment:
```suggestion
    While this is user configurable, by default data is gathered every 5, 10 or 60 minutes. In case Query Analytics is enabled and SQL query examples are gathered, we don't use any special processing for personal or confidential data. PMM server has no clue about the meaning of the data inside the SQL query. 
```

Reply:
if we are talking about QAN, it's not configurable.